packages/react-core/src/components/Masthead/MastheadBrand.tsx

```diff
@@ -15,10 +15,19 @@ export interface MastheadBrandProps
 export const MastheadBrand: React.FunctionComponent<MastheadBrandProps> = ({
   children,
   className,
-  component = 'a',
+  component,
   ...props
 }: MastheadBrandProps) => {
-  const Component = component as any;
+  let Component = component as any;
+  if (!component) {
+    if (props?.href !== undefined) {
+      Component = 'a';
+    } else if (props?.onClick !== undefined) {
+      Component = 'button';
```

Comment:
Looks like we don't support styling for this as a `<button>` - just want to make sure that's a valid use case before filing an issue for it. Also if it's a button, should it have `type="button"`?

Reply:
Removing it for now, especially if we don't currently have styling for it.